src/App.tsx

```diff
@@ -1,119 +1,19 @@
-//         id: uuidv4(),
-//     },
-//     {
-//         title: "France",
-//         body:
-//         "Is a transcontinental country spanning Western Europe and several overseas regions and territories.",
-//         id: uuidv4(),
-//     },
-// ];
-
-const App = () => {
-    const [data, setData] = useState<CardsDataDTO[]>([]);
-    const [isEdit, setIsEdit] = useState(false);
-    const [isOpen, setIsOpen] = useState(false);
-    const [isLoading, setIsLoading] = useState(false);
-
-    useEffect(() => {
-        getData()
-    }, [])
-
-    async function getData(){
-        setIsLoading(true);
-        await  fetch('https://jsonplaceholder.typicode.com/posts')
-        .then(response => response.json())
-        .then(data => setData(data))
-        setIsLoading(false);
-    }
-
-
-    data.forEach(x=>{
-        x.oldTitle = x.title;
-        x.oldDescription = x.body;
-    })
-
-    const addCardHandler = async(title: string, text: string) => {
-        let newData={
-            title: title,
-            body: text,
-            id: uuidv4()
-        }
-
-        // setData((prevData) => {
-        // return [...prevData, newData];
-        // });
-
-        let response = await fetch('https://jsonplaceholder.typicode.com/posts', {
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json'
-            },
-            body: JSON.stringify(newData)
-
-        })
-
-        let result = await response;
-        console.log(result.status );
-    };
-
-    const removeCardHandler = (id: string) => {
-        let filteredData = data.filter((el) => el.id !== id);
-
-        setData(filteredData);
-    };
-
-    const editCardHandler = ( ) => {
-
-        setIsEdit(!isEdit);
-
-    };
-
-    const toggleModal = () => {
-        setIsOpen(!isOpen);
-    };
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import Cards from './components/Card/Cards';
+import MainPage from './components/MainPage/MainPage';
 
+function App() {
     return (
-        <div className="App">
-        <Header />
-        <div className={isOpen ? "backdrop" : ""} onClick={toggleModal}></div>
-        <div>
-            <button className="btn-add" onClick={toggleModal}>Add Card</button>
-        </div>
-        {isOpen ? <Modal addCardHandler={addCardHandler} closeModal={toggleModal}/> : null}
-        {isLoading ? <p>Loading...</p> : <Card
-            data={data}
-            removeCardHandler={removeCardHandler}
-            editCardHandler={editCardHandler}
-            isEdit={isEdit}
-        />}
-        </div>
+        <BrowserRouter>
+            <div className = "App">
+                <Switch>
+                    <Route exact path = '/' component = {MainPage} />
+                    <Route path = '/main' component = {MainPage} />
+                    <Route path = '/cards' component = {Cards} />
+                </Switch>
```

Comment:
Please also add some kind of '404' page to handle all unknown paths.

Reply:
done